packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -19,3 +30,72 @@ abstract class Screen {
   /// Builds the body to display for this tab.
   Widget build(BuildContext context);
 }
+
+enum DevToolsScreenType {
+  inspector,
+  timeline,
+  memory,
+  performance,
+  logging,
+  info,
+  connect,
+  debugger,
+  network,
+  simple,
+}
+
+extension DevToolsScreenTypeExtension on DevToolsScreenType {
+  static DevToolsScreenType fromId(String id) {
+    switch (id) {
+      case 'inspector':
```

Comment:
you can perform this mapping without hardcoding it. see the similar code albert wrote for various flex enums in the layout explorer. That way we won't accidentally have a typo serializing and deserializing the ids.

Reply:
Not finding the examples you are referencing. Could you provide a link?

Reply:
see EnumUtils

Reply:
done